init.sqf

```diff
@@ -1,21 +1,16 @@
+// Enregistrement des commandes de requête Zeus
+["zeus", {(_this select 0) spawn GSRI_fnc_handleRequest}, "all"] call CBA_fnc_registerChatCommand;
+
 //EOS SYSTEM
```

Comment:
What is this ?

Reply:
Used to register the `zeus `command in IG chat. So you can write `#zeus <command>` and the handler will take care of it.

Reply:
My point was line 4 : EOS SYSTEM

Reply:
Out-of-the-box occupation system for red-colored grid zones. EOS = Ennemy Occupation System.